### PR TITLE
Fix xcversion deps / bundler cyclic dependency error

### PR DESCRIPTION
**Pull request by milch**
Addresses #7573

### CODE REVIEW

**Review by mfurtak — APPROVED** [review]

File: fastlane/lib/fastlane/actions/xcode_install.rb [hunk 1]
```diff
@@ -9,7 +9,12 @@ def self.run(params)
         ENV["XCODE_INSTALL_USER"] = params[:username]
         ENV["XCODE_INSTALL_TEAM_ID"] = params[:team_id]
 
-        require 'xcode/install'
+        begin
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/xcode_install.rb]: We should use `Actions.verify_gem!('cocoapods')` 👍

File: fastlane/lib/fastlane/helper/xcversion_helper.rb [hunk 2]
```diff
@@ -4,7 +4,12 @@ class XcversionHelper
       def self.find_xcode(req)
         req = Gem::Requirement.new(req.to_s)
 
-        require 'xcode/install'
+        begin
+          require 'xcode/install'
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/helper/xcversion_helper.rb]: Same here

Comment (KrauseFx) [inline fastlane/lib/fastlane/helper/xcversion_helper.rb]: oh https://github.com/fastlane/fastlane/pull/7554

**Comment by milch** [comment]
Closing in favour of #7554

**Pull request closed without merging**